.github/workflows/python-package.yml

```diff
@@ -0,0 +1,47 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
```

Comment:
I'm assuming we would just add `v2` to this list when we port it to that branch?

Reply:
We would change it to just v2 for the v2 branch.